### DISCUSSION TITLE
Is backpack meant to show error messages on 500 pages?

### DISCUSSION THREAD

**Discussion by Jamesking56** [Laravel-Backpack/community-forum#303] [Q&A (Help)] [answered]
Hi everyone,

When an error in the code occurs, Laravel Backpack displays an error 500 screen.

Lately, I've been seeing detailed PHP stack traces on this screen, even though `APP_DEBUG` is set to `false`.

Is this meant to happen? Is there a way to turn this off as it currently shows detailed errors on my client's demo.

Cheers.

**Comment by tabacitu** [comment, marked as answer]
The default error screens for Backpack do include one line with the error message, but never entire stack traces. That's probably powered by Whoops or Flare, depending on your Laravel version. So it's a Laravel-thing, not a Backpack-thing.

Maybe your .env or config variables are cached and Laravel picks up APP_DEBUG as `true`? 

```
php artisan config:clear
```

**Reply by Jamesking56** [reply]
Its the one-liner I'm talking about, is that a leak of info or do you think its safe for production?

**Reply by tabacitu** [reply]
Ah, got it. We consider it safe for production, that's why we included it. It solves one problem we had with clients, where they saw errors and said "I got error 500", which... tells us nothing. This way, they can at least say the first line, which will provide context.

If you don't like it, you can easily get rid of it - just remove the `resources/views/errors` directory, or change the blade files to suit your needs. If you delete them, it'll fall back to the default Laravel error views, which are prettier but hide that line.

Hope it helps!

**Reply by Jamesking56** [reply]
Thanks yeah that helps a lot :+1:
